src/coreclr/inc/eetwain.h

```diff
@@ -455,6 +458,9 @@ virtual
 bool IsGcSafe(  EECodeInfo     *pCodeInfo,
                 DWORD           dwRelOffset);
 
+virtual
+bool InterruptibleSafePointsEnabled();
```

Comment:
Nit: It would be fine to this to be non-virtual (static) EECodeManager::InterruptibleSafePointsEnabled method.

The ICodeManager abstraction enabled multiple fundamentally different GCInfo encodings at one point. We have just one GCInfo encodings these days, so the abstraction is not very useful and bit-rotting.

Since the InterruptibleSafePoints are specific to one GC encoding, it makes more sense to have that config on the specific EECodeManager.

Reply:
Yes, could be static.